docs/framework-docs.md

```diff
@@ -0,0 +1,327 @@
+## Provisioning
+- Packaging (http server -> Lambda function)
+- Static assets: icons only
+  - Serving them out of S3
+  - Validate the contents to be harmless icons??
+
```

Comment:
The URL of the assets is `$SITE_URL/plugins/com.mattermost.apps/api/v1/static/$APP_ID$/$ASSET_NAME`